example/test/lint_test.bats

```diff
@@ -42,7 +48,7 @@ EOF
 }
 
 @test "should produce reports" {
-	run $BATS_TEST_DIRNAME/../lint.sh //src:all
+	run $BATS_TEST_DIRNAME/../lint.sh //src:all --no@aspect_rules_lint//lint:color
```

Comment:
Bazel supports negating custom flags with `--no<flag>` ?

Reply:
I was surprised too - yes I tested it!